docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md

```diff
@@ -0,0 +1,268 @@
+# Single New EKS Cluster Open Source Observability Accelerator for Neuron-based clusters
+
+[AWS Trainium](https://aws.amazon.com/machine-learning/trainium/) and [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/) are accelerated Machine Learning (ML) chips (or ML accelerators), designed and built by AWS. They are also referred to as Neuron Devices. Each Neuron device includes multiple NeuronCores, the machine learning compute cores.
+
+Amazon EC2 ML instances belong to the [Trn1/Trn1n](https://aws.amazon.com/ec2/instance-types/trn1/), [Inf1](https://aws.amazon.com/ec2/instance-types/inf1/) and [Inf2](https://aws.amazon.com/ec2/instance-types/inf2/) families. Trn1/Trn1n instances feature multiple AWS Trainium accelerators and support high-performance training. Inf1 and Inf2 instances feature multiple AWS Inferentia accelerators and support high-performance and low-latency inference.
+
+[AWS Neuron](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/) is an SDK with a compiler, runtime, and profiling tools that helps developers deploy models on both AWS Inferentia accelerators and train them on AWS Trainium accelerators. It integrates natively with popular ML frameworks, such as PyTorch and TensorFlow.
```

Comment:
There term accelerators is unclear here

Reply:
it is explained at the beginning of the doc: "[AWS Trainium]and [AWS Inferentia] are accelerated Machine Learning (ML) chips (or ML accelerators)", see also here: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/glossary.html?highlight=glossary#neuron-devices-accelerated-machine-learning-chips